webui/react/src/components/kit/InlineForm.tsx

```diff
@@ -0,0 +1,87 @@
+import { Form } from 'antd';
+import React, { useCallback, useState } from 'react';
+
+import Button from 'components/kit/Button';
+import Icon from 'components/kit/Icon';
+
+import css from './InlineForm.module.scss';
+
+interface Props {
+  label: string;
+  inputValue?: string | number;
```

Comment:
Value can be anything the input accepts/emits which is more than string and number.